hil-test/README.md

```diff
@@ -0,0 +1,108 @@
+# hil-test
+
+Hardware-in-loop testing for `esp-hal`.
+
+For assistance with this package please [open an issue] or [start a discussion], or contact @jessebraham on Matrix.
```

Comment:
We should remove having a specific contact, in favour of sharing the load between us all.

Reply:
Definitely, also slipped in!